content/docs/run-api-server/running.mdx

```diff
@@ -47,20 +48,20 @@ Horizon provides most of its utility through ingested data, and your Horizon ser
 **We highly recommend having more than one ingesting instance**, as this makes it easier to avoid downtime during upgrades and adds resilience to your infrastructure, ensuring you always have the latest network data.
 
 ### Ingesting Live Data 
-To **enable** ingestion, you must either pass `--ingest=true` on the command line or set the `INGEST` environment variable to "true". Note that you can have multiple ingesting machines in your cluster.
+To **enable** ingestion, you must either pass `--ingest=true` on the command line or set the `INGEST=true` in your environment. Again, note that you can have multiple ingesting machines in your cluster.
 
 ### Ingesting Historical Data
-To ingest *historical* data from Stellar Core, you need to run `horizon db reingest range <start> <end>`. If you're running a [full validator](../run-core-node/index.mdx#full-validator) with published history archive(s), for example, you might want to ingest the network's entire history. You can run this process in the background while your Horizon server is up. This continuously decrements the `history.elder_ledger` in your `/metrics` endpoint until `NUM_LEDGERS` is reached and the backfill is complete.
+To ingest *historical* data from the Stellar network, you need to run Horizon `db reingest range <start> <end>` command. If you're running a [full validator](../run-core-node/index.mdx#full-validator) with published history archive(s), for example, you might want to ingest the network's entire history. You can run this process in the background while your Horizon server is up. This continuously decrements the `history.elder_ledger` in your `/metrics` endpoint until `NUM_LEDGERS` is reached and the backfill is complete.
 
 ### Reingesting Ledgers
-To reingest older ledgers—occasionally necessary after a version upgrade—or to ingest ledgers closed by the network before you started Horizon use the `horizon db reingest range <start> <end>` command:
+To reingest older ledgers—occasionally necessary after a version upgrade—or to ingest ledgers closed by the network before you started Horizon use the `db reingest range <start> <end>` command:
 
 <CodeExample>
 
 ```
-horizon1> horizon db reingest range 1 10000
-horizon2> horizon db reingest range 10001 20000
-horizon3> horizon db reingest range 20001 30000
+horizon1> stellar-horizon db reingest range 1 10000
+horizon2> stellar-horizon db reingest range 10001 20000
+horizon3> stellar-horizon db reingest range 20001 30000
```

Comment:
Maybe we can link to the doc in which there's a detailed explanation of pubnet history reingestion? Can be helpful.